docs/Sonarr/V3/Sonarr-Release-Profile-RegEx-Anime.md

```diff
@@ -178,11 +178,9 @@ Add this to your Preferred with a score of **[0]**. This is for renames only and
 /(dual[ ._-]?audio)/i
 ```
 
-#### Optional (uncut|unrated|uncensored)
+#### Uncut|Unrated|Uncensored
```

Comment:
Nit: I would just say "Uncensored" here. uncut/unrated aren't really terms I see used in the anime community. This isn't a tooling requirement, just a human readability concern.